### PR TITLE
2023 - Fixed cut off label in extra small input field

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixed an issue where the colorpicker label is cut off in extra small input field.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/2023

**Steps necessary to review your pull request (required)**:
- Pull this branch
- Run http://localhost:4000/components/colorpicker/example-sizes.html
- The text in `Extra Small` input field should be in ellipsis.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 - `[App Menu]` Fixed some accessibility issues on the nav menu. ([#1721](https://github.com/infor-design/enterprise/issues/1721))
 - `[Busy Indicator]` Fixed a bug that causes a javascript error when the busy indicator is used on the body tag. ([#1918](https://github.com/infor-design/enterprise/issues/1918))
+- `[Colorpicker]` Fixed an issue where the colorpicker label is cut off in extra small input field. ([#2023](https://github.com/infor-design/enterprise/issues/2023))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]:
```suggestion
- `[Colorpicker]` Fixed an issue where the colorpicker label is cut off in extra small input field. ([#2023](https://github.com/infor-design/enterprise/issues/2023))
```

Comment (tmcconechy) [reply docs/CHANGELOG.md]: Removed trailing space (lint error)

**Comment by clepore** [comment]
Hey @ericangeles , since we needed to get an RC release out to QA today, we switch the base of this to master, so no rush 👍

**Comment by tmcconechy** [comment]
@clepore that redbasing didnt work so good. maybe have to redo this to master.

**Comment by tmcconechy** [comment]
@ericangeles unfortunately will have to repush the fix on a fresh branch off master. Sorry about that.
I can update the baselines for you for the tests.

**Comment by clepore** [comment]
I merged 4.18.x into master and then pulled master into this branch. I didn't see any conflicts or anything, but @ericangeles, take a look and see.

**Comment by tmcconechy** [comment]
Just seeing a lot of changed here https://github.com/infor-design/enterprise/pull/2044/files was only a one liner of a fix originally

**Comment by clepore** [comment]
https://github.com/infor-design/enterprise/compare/2023-colorpicker-label-cut-off

**Pull request closed without merging**